import-dashboards.py

```diff
@@ -160,10 +160,30 @@ def delete_api_key(db_key, upgrade):
     con.close()
 
 
-def add_datasources(api_key):
-    r = requests.get('%s/api/datasources' % (HOST,), headers=grafana_headers(api_key))
-    print ' * Datasources: %r %r' % (r.status_code, r.content)
-    ds = [x['name'] for x in json.loads(r.content)]
+def _add_metrics_datasource(api_key, ds):
+    if 'Metrics' not in ds:   # https://jira.percona.com/browse/PMM-6518
+        print ' * Adding Metrics Data Source'
+        data = json.dumps({'name': 'Metrics', 'type': 'prometheus', 'jsonData': {'keepCookies': [], 'timeInterval': '1s', 'httpMethod': 'POST'}, 'url': 'http://127.0.0.1:9090/prometheus/', 'access': 'proxy', 'isDefault': True})
+        r = requests.post('%s/api/datasources' % HOST, data=data, headers=grafana_headers(api_key))
+        print r.status_code, r.content
+        if r.status_code != httplib.OK:
+            print ' * Cannot add Metrics Data Source'
+            sys.exit(-1)
+    else:
+        print ' * Modifing Metrics Data Source'
+        r = requests.get('%s/api/datasources/name/Prometheus' % (HOST,), headers=grafana_headers(api_key))
```

Comment:
Pretty sure that should not be `Prometheus`

Reply:
We are not switching to Victoria metrics yet.

![Screenshot_20201012_131957](https://user-images.githubusercontent.com/25184101/95735683-cf228480-0c8d-11eb-8af3-9208d465166f.png)

Reply:
I understand that. But in this code you get Prometheus configuration (see URL)